tables/lg-ug-g1.utb

```diff
@@ -0,0 +1,58 @@
+# liblouis: Luganda Grade 1 Braille
+#
+#-display-name: Luganda Grade 1
+#-index-name: Luganda Grade 1
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#
+#+language: lg
+#+region: ug
+#+type: literary
+#+contraction: no
+#+grade: 1
+#+dots: 6
+#+direction: both
+#
+# Copyright (C) 2022 Compass Braille, UK, https://compassbraille.org
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# This table was prepared using feedback from Ugandan proof readers of the
+# Luganda Bible 1968 Edition (Bible Society of Uganda, 2013) in Braille.
+#
+# Luganda uses the same 26 latin letters as English plus the letter Ŋŋ (LATIN
+# LETTER ENG: U+014A and U+014B). It uses the same rules as UEB except for
+# this letter. In UEB it is 45-1345. In Luganda it usually appears as a pair
+# but sometimes singly and in both cases it is transcribed as 1345-1245 - the
+# same as the letters "ng". Backtranslation is therefore problematic but there
+# seem to be predicatble patterns as to where and how it is used so that this
+# table works well in both directions in most cases.
+
+
+# Ŋŋ - U+014A and U+014B
+noback always ŋ 1345-1245 # ng
+noback always ŋŋ 1345-1245 # ng
```

Comment:
Maybe these should be character definitions. Have you tried if that works?

Reply:
I have defined Ŋ and ŋ.
(Incidently using "base uppercase" didn't work - Ŋ would be based on the UEB definition - but perhaps it depends on the order in which the lines appear?)

Reply:
Tests for capital letter added.

Reply:
I think if you move the `lowercase Ŋ` *after* the `include` it should work without the `uppercase Ŋ` (and a `base uppercase Ŋ` isn't needed either because the UEB table already contains it).

Reply:
This also works (and fits with my theory) but seems not so good I'm sure you'll agree:
```
lowercase ŋ 1345-1245
include en-ueb-g1.ctb
lowercase ŋ 1345-1245
base uppercase Ŋ ŋ
```

Reply:
Ah yes, your observations are correct. Note that you don't need the `base uppercase Ŋ ŋ` rule. I would argue both solutions are equally ugly. They contain an equal amount of duplication. Anyway, you may choose how you implement it. As long as it is covered with test and it works, it is fine.